AnnoyancesFilter/Popups/sections/push-notifications_specific.txt

```diff
@@ -103,7 +103,7 @@ kojikisokuhou.com##.notification
 ||nicoblomaga.jp/material/31/tatefication.png$domain=kojikisokuhou.com
 ||thetennisdaily.jp/assets/js/push-modal.js
 ||ozmall.co.jp/scripts/js/user-notification.js
-/notification.js$domain=jauns.lv|a2zapk.com|rt.com
+/notification.js$domain=jauns.lv|rt.com
```

Comment:
Where do you need this rule and for what?

Reply:
To resolve [this issue](https://github.com/AdguardTeam/AdguardFilters/issues/133898#issue-1433833432).

If you mean literally `/notification.js$domain=...`, it was mentioned at https://github.com/AdguardTeam/AdguardFilters/commit/b73abe8ecac7c7b6511a4f9f7d663b8bf60308bf and https://github.com/AdguardTeam/AdguardFilters/commit/b73abe8ecac7c7b6511a4f9f7d663b8bf60308bf.

Reply:
I see that images are blocked by this rule until you scroll the page a bit. Got it.